unity/Assets/Scripts/DroneFPSAgentController.cs

```diff
@@ -116,7 +116,7 @@ public class DroneFPSAgentController : BaseFPSAgentController {
         }
 
         // generates object metadata based on sim object's properties
-        public override ObjectMetadata ObjectMetadataFromSimObjPhysics(SimObjPhysics simObj, bool isVisible) {
+        public override ObjectMetadata ObjectMetadataFromSimObjPhysics(SimObjPhysics simObj, bool isVisible, bool isInteractable) {
```

Comment:
`isInteractable` doesn't seem to be used below?

Reply:
this has been added to drone metadata